Extensions/people_notifier.js

```diff
@@ -12,6 +12,7 @@ XKit.extensions.people_notifier = new Object({
 	apiKey: "fuiKNFp9vQFvjLNvx4sUwti4Yb5yGutBN4Xh10LXZhhRKjWlV4",
 
 	blogs: [],
+	searchable_blogs: {},
```

Comment:
I have a feeling we can maintain this functionality while only using the existing `blogs` data structure.